sechub-commons-model/src/test/java/com/mercedesbenz/sechub/commons/model/SecHubRemoteCredentialSupportTest.java

```diff
@@ -0,0 +1,181 @@
+package com.mercedesbenz.sechub.commons.model;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.io.File;
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.EmptySource;
+import org.junit.jupiter.params.provider.NullSource;
+
+import com.mercedesbenz.sechub.test.TestFileReader;
+
+class SecHubRemoteCredentialSupportTest {
+
+    private SecHubRemoteCredentialSupport supportToTest;
+
+    @BeforeEach
+    void beforeEach() {
+        supportToTest = new SecHubRemoteCredentialSupport();
+    }
+
+    @Test
+    void read_simple_credential_configuration_from_json_string() {
+        /* prepare */
+        String json = TestFileReader.loadTextFile(new File("./src/test/resources/sechub_remote_credentials_simple_config.json"));
+
+        /* execute */
+        SecHubRemoteCredentialConfiguration configuration = supportToTest.getRemoteCredentialConfigurationFromJSONString(json);
+
+        /* test */
+        List<SecHubRemoteCredentialContainer> credentials = configuration.getCredentials();
+        assertEquals(1, credentials.size());
+
+        SecHubRemoteCredentialContainer container = credentials.get(0);
+        assertNotNull(container);
+        assertEquals("example-user", container.getUser());
+        assertEquals("example-api-token-or-password", container.getPassword());
+        assertEquals(".*github.com/*", container.getRemotePattern());
```

Comment:
It is just an example, but it must be clear that we have here regular expression here. So it should be:
```suggestion
        assertEquals(".*github.com/.*", container.getRemotePattern());
```